lips/lip-6.mdx

```diff
@@ -53,7 +52,7 @@ resource struct ValidatorConfig {
   human_name: vector<u8>,
 }
 ```
-It contains a `human_name` of the account which is set at the account creation time and cannot be changed. The `config` and `operator_account` fields are initially set to `none`. The `config` is a place holder for parameters of the Validator Node.
+It contains a `human_name` of the account, a UTF-8 encoded string, which is set at the account creation time and cannot be changed. The `config` and `operator_account` fields are initially set to `none`. The `config` is a place holder for parameters of the Validator Node.
```

Comment:
nit: typo a few paragraphs down "nextn" -> "next" (sorry, I couldn't add the comment to the actual line for some reason :/)

Reply:
nit: another typo (many paragraphs down) "LibraAccount" -> "Libra Account" ?

Reply:
probably shouldn't have done this edit in github directly....